code/modules/research/xenobiology/xenobiology.dm

```diff
@@ -716,6 +716,7 @@
 			to_chat(SM, span_userdanger("You also become depressingly aware that you are not a real creature, but instead a holoform. Your existence is limited to the parameters of the holodeck."))
 		to_chat(user, span_notice("[SM] accepts [src] and suddenly becomes attentive and aware. It worked!"))
 		SM.copy_languages(user)
+		SM.faction = list("neutral")
```

Comment:
you should include the mob's reference as a faction as well, all living mobs have a faction to themselves!

https://github.com/tgstation/tgstation/blob/c10e1e60db4bfba3e9c0ca543a24223a37a25304/code/modules/mob/living/living.dm#L1-L10

```suggestion
		SM.faction = list([REF(src)], "neutral")
```